modules/apps/object/object-web/types/src/main/resources/META-INF/resources/js/components/ObjectView/ModalAddDefaultSortColumn/ModalAddDefaultSortColumn.d.ts

```diff
@@ -13,10 +13,18 @@
  */
 
 import React from 'react';
-import './ModalAddColumnsObjectCustomView.scss';
 interface IProps extends React.HTMLAttributes<HTMLElement> {
+	editingObjectFieldName: string;
+	header: string;
+	isEditingSort: boolean;
 	observer: any;
 	onClose: () => void;
 }
-declare const ModalAddColumnsObjectCustomView: React.FC<IProps>;
-export default ModalAddColumnsObjectCustomView;
+export declare function ModalAddDefaultSortColumn({
+	editingObjectFieldName,
+	header,
+	isEditingSort,
+	observer,
+	onClose,
+}: IProps): JSX.Element;
+export {};
```

Comment:
Why are you exporting an empty object here?

Reply:
I really don't know, this file is generated automatically :laughing:

Reply:
Oh ok, sorry about that